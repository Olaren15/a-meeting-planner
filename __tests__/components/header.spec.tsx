import { render } from "@testing-library/react";
import Header from "@/components/header";

describe("Header", () => {
    it("Should render", () => {
        const { container } = render(<Header />);

        expect(container).toMatchSnapshot();
    });
});
