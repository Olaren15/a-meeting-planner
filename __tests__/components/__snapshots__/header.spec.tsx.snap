// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render 1`] = `
<div>
  <header
    class="flex flex-col gap-1 bg-neutral-600 p-4 text-center text-white"
  >
    <h1
      class="text-3xl"
    >
      A Meeting Planner
    </h1>
    <h2>
      The boring way to plan your meetings
    </h2>
  </header>
</div>
`;
